File: Back/src/TesteConfitec.Domain/Usuario.cs
using System;
using System.Collections.Generic;
using TesteConfitec.Domain.Enum;

namespace TesteConfitec.Domain
{
    public class Usuario
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Sobrenome { get; set; }
        public string Email { get; set; }
        public DateTime DataNascimento { get; set; }
        public Escolaridade EscolaridadeId { get; set; }
        public IEnumerable<HistoricoEscolar> HistoricosEscolares { get; set; }
    }
}

File: Front/TesteConfitec-App/src/app/models/Usuario.ts
import { Escolaridade } from "./Escolaridade";
import { HistoricoEscolar } from "./HistoricoEscolar";

export interface Usuario {
  id: number;
  nome: string;
  sobrenome: string;
  email: string;
  dataNascimento?: Date;
  historicoEscolar: HistoricoEscolar[];
  escolaridadeId: Escolaridade;
}
